### ISSUE TITLE
1.7.10 changes to the configs don't apply

### ISSUE THREAD

**Issue by zpazz** [Malorolam/LootBags#189]
as stated above. i re-installed the mod+config and only swapped the text colors of the lootbags for testing reasons, and even that did not apply although the configs were properly saved.

**Comment by zpazz** [comment]
getting into detail: apparently the config does get saved. the command '$CRAFTEDFROM:$NULL' and drop customization applies. but any changes made to $ISSECRET $BAGTEXTCOLOR and others ... just have no effect. also adding a new custom lootbag won't show up in the NEI menu despite having set it to non-secret... i don't know where to start looking for what causes this.

**Comment by zpazz** [comment]
okay customizing the drops of the lootbags works as well.

**Comment by Malorolam** [comment]
1.7.10 is not under development anymore. If you post your bag config here I'll look at it.

**Comment by zpazz** [comment]
thanks for the quick reply. if i change lets say $BAGTEXTCOLOR to ''BLUE'' nothing will happen. same goes with the displayed texts (unopened/opened/shift)
![11](https://user-images.githubusercontent.com/27631439/60898650-76359680-a269-11e9-9dab-d02d49ce0914.JPG)

**Comment by Malorolam** [comment]
Are you putting $BLUE or BLUE? The $ is important so the parser interprets commands correctly.

**Comment by zpazz** [comment]
yep. i also simply edited the bagtext slightly and nothing changed, while other changes did apply (craftedfrom/whitelist...)

**Comment by Malorolam** [comment]
That seems odd, as the $BLUE for the other bag clearly works. Did you try copy and pasting one of the existing lines in place? Also are you restarting the game each time?

**Comment by zpazz** [comment]
we're hosting at nitrado so i restarted, tested creating a new world etc. but the changes to the color / open-and shift text did not apply. i really double tested it. i did not test it standalone tho so it could be a conflict with one of the other mods - or those lines are client side only?

**Comment by Malorolam** [comment]
The colours are interacted with on client-side code, so even though it isn't formally client-side, it only does something on the client.

**Comment by zpazz** [comment]
so if i change the server config but not the config in my client files there won't be any changes to the colors/text? i mean getting the drops right was my main concern, if adjusting those cosmetics doesn't work thats not really a big deal.

**Comment by Malorolam** [comment]
Well, you should make sure that the configs are the same for the server and your players for all mods. Otherwise you encounter desync issues. Forge handles the configs generated by its system, but as the bag config is custom it doesn't apply there.

**Comment by zpazz** [comment]
okay i will take your advice haha. i'm only using your mod on our little private server tho - it perfectly adds to dungeon mods like chocolate/roguelike/infernalmobs etc. and we love it. thank you for replying so fast :)
